### PR TITLE
Sending notifications

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-28958, https://jira.ez.no/browse/EZEE-1846
| Versions      | 2.1, master

Replaces the old (v1) version of "Flex Workflow notifications" with one working for v2, together with new info about displaying notification bars from https://jira.ez.no/browse/EZP-28958

### CODE REVIEW

**Comment by DominikaK** [comment]
Ping @Nattfarinn for review, also /cc @bdunogier

File: docs/cookbook/sending_notifications.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Sending notifications
+
+You can send two types on notifications to the users.
+
+## Display notification bars
+
+You can have your PHP code send notification that will be displayed as a message bar in the Back Office.
+To do that, inject the `NotificationHandlerInterface` into your class.
+You can use one of the four methods representing four notification types: `info`, `success`, `warning` and `error`.
```
Comment (bdunogier) [inline docs/cookbook/sending_notifications.md]: I'd remove the second "four".

File: docs/cookbook/sending_notifications.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+``` php
+$this->notificationHandler->info('Notification text');
+```
+
+To have the notification translated, make use of the `TranslatorInterface`.
+You need to provide the message strings in the translation files under the correct domain and key.
+
+``` php
+$this->notificationHandler->info(
+    $this->translator->trans(
```
Comment (bdunogier) [inline docs/cookbook/sending_notifications.md]: Should we add the phpdoc comment (`@desc`) that provides the source text for extraction ?

File: docs/cookbook/sending_notifications.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+```
+
+![Example of an info notification](img/notification2.png)
+
+## Create custom notifications using the Flex Workflow mechanism
+
+!!! enterprise
+
+    You can send your own custom notifications to the user
+    with the same mechanism that is used to send notification from Flex Workflow.
```
Comment (bdunogier) [inline docs/cookbook/sending_notifications.md]: How does the whole chapter differ from the one above ?

Comment (DominikaK) [reply docs/cookbook/sending_notifications.md]: You're right, I've added a clarification with screenshots.

**Pull request merged**